apps/rehabstod/src/store/api.ts

```diff
@@ -123,11 +123,12 @@ export const api = createApi({
       }),
       providesTags: ['SickLeaveSummary'],
     }),
-    getSickLeavePatient: builder.query<Patient, { encryptedPatientId: string }>({
-      query: ({ encryptedPatientId }) => ({
+    getSickLeavePatient: builder.query<Patient, { encryptedPatientId: string | undefined }>({
```

Comment:
We should probably not allow fetching with undefined `encryptedPatientId`. Please revert this change.